kano_qt_apps/kano_qt_apps.pro

```diff
@@ -1,5 +1,7 @@
 TEMPLATE = subdirs
 
+QT += qml
```

Comment:
Also don't think this is needed. It's going to be added by `kano-qt-sdk`

Reply:
Agreed.